src/sql/cmd/columns.py

```diff
@@ -0,0 +1,17 @@
+from sql import inspect
+from sql.util import sanitize_identifier
+from sql.cmd.cmd_utils import CmdParser
+
+
+def execute_columns_command(others):
+    """
+    Execution logic for the columns command
```

Comment:
A more descriptive docstring (also applicable for the others)

```suggestion
    Implementation of `%sqlcmd columns`
```